.github/workflows/pytest-core-nompi.yml

```diff
@@ -29,65 +29,72 @@ jobs:
            pytest-ubuntu-py37-gcc5-omp,
            pytest-ubuntu-py38-gcc6-omp,
            pytest-ubuntu-py36-gcc7-omp,
-           pytest-ubuntu-py37-gcc7-noomp,
+           pytest-ubuntu-py310-gcc10-omp,
            pytest-ubuntu-py38-gcc8-omp,
            pytest-ubuntu-py39-gcc9-omp,
            pytest-osx-py37-clang-omp,
-           pytest-docker-py36-gcc-omp
+           pytest-docker-py37-gcc-omp
         ]
         set: [base, adjoint]
         include:
         - name: pytest-ubuntu-py37-gcc5-omp
-          python-version: 3.7
+          python-version: '3.7'
           os: ubuntu-18.04
           arch: "gcc-5"
           language: "openmp"
           sympy: 1.8
 
         - name: pytest-ubuntu-py38-gcc6-omp
-          python-version: 3.8
+          python-version: '3.8'
           os: ubuntu-18.04
           arch: "gcc-6"
           language: "openmp"
           sympy: 1.7
 
         - name: pytest-ubuntu-py36-gcc7-omp
-          python-version: 3.6
+          python-version: '3.6'
           os: ubuntu-18.04
           arch: "gcc-7"
           language: "openmp"
           sympy: 1.8
 
-        - name: pytest-ubuntu-py37-gcc7-noomp
-          python-version: 3.7
-          os: ubuntu-18.04
-          arch: "gcc-7"
+        - name: pytest-ubuntu-py310-gcc10-omp
+          python-version: '3.10'
+          os: ubuntu-20.04
+          arch: "gcc-10"
           language: "C"
-          sympy: 1.8
+          sympy: 1.9
 
         - name: pytest-ubuntu-py38-gcc8-omp
-          python-version: 3.8
+          python-version: '3.8'
           os: ubuntu-18.04
           arch: "gcc-8"
           language: "openmp"
           sympy: 1.9
 
         - name: pytest-ubuntu-py39-gcc9-omp
-          python-version: 3.9
+          python-version: '3.9'
           os: ubuntu-20.04
           arch: "gcc-9"
           language: "openmp"
           sympy: 1.8
 
+        - name: pytest-ubuntu-py310-gcc10-omp
```

Comment:
adding a new one, but not dropping an old one? is this PR replacing a job, or adding a new one?

Reply:
We discused in the meeting yesterday to maybe replace a 3.7 job, ` ' ' ` are needed from now for github actions.

Reply:
https://github.com/actions/setup-python/issues/160